resources/README.md

```diff
@@ -0,0 +1,9 @@
+# Resources
+
+A repository for various courses, guides, blog posts that we find useful for our processes.
+
+## Courses
+
+### Rails Speed
+
+* [Rails Speed](https://www.railsspeed.com/): A full-stack course that gives you the tools to make Ruby on Rails applications faster, more scalable, and simpler to maintain. Look for it in Subvisual's Google Drive directory, or ask [@naps62](mailto:miguel@subvisual.co) if you need help finding it.
```

Comment:
that was on purpose. Is there any harm in making it public that we have one? As long as we don't publish links to those directories (which are still protected by authentication, but still...)